src/executors/ExecutorTeamCity.js

```diff
@@ -1,8 +1,8 @@
-//import { default as TeamCity } from 'teamcity-rest-api';
+import { default as TeamCity } from 'teamcity-rest-api';
 import { Executor } from '../agent/Executor';
-import { AgentUtils } from '../agent/AgentUtils';
+import { Utils } from '../agent/AgentUtils';
 import { default as json } from 'format-json';
-let log = AgentUtils.logger();
+let log = Utils.logger();
 
```

Comment:
we have started adding sample .githubTaskManager.json config as a comment at the top of each executor.  can you please do same?  will likely jsdoc each, so you could do that instead if you like.